src/Init/System/IO.lean

```diff
@@ -455,6 +456,16 @@ see there for more information.
 -/
 @[extern "lean_io_rename"] opaque rename (old new : @& FilePath) : IO Unit
 
+/--
+Creates a temporary file in the most secure manner possible. There are no race conditions in the
+file’s creation. The file is readable and writable only by the creating user ID. Additionally
+on UNIX style platforms the file is executable by nobody. The function returns both a `Handle`
+to the already opened file as well as its `FilePath` in case that is of interest.
+
+Note that it is the caller's job to remove the file after use.
```

Comment:
I find “in case that is of interest” a bit confusing when the next line tell me I have to delete the file, so it should be of interest to every caller? 

Maybe just drop that half-sentence:

```suggestion
to the already opened file as well as its `FilePath`.

Note that it is the caller's job to remove the file after use.
```